docs/guides/contributing.md

```diff
@@ -374,28 +374,25 @@ GitHub has additional documentation on [creating a pull request from a fork](htt
 
 [DoneJS.com](https://donejs.com/) is hosted on [GitHub pages](https://pages.github.com/) from the [`gh-pages`](https://github.com/donejs/donejs/tree/gh-pages) branch.
 
-First, from within your local DoneJS repository, create a new clone in the `site` folder and install the dependencies:
+First, from within your local DoneJS repository, make sure you have the latest changes and don’t have a local `gh-pages` branch:
 
 ```shell
-git clone git@github.com:donejs/donejs.git -b gh-pages site/
```

Comment:
I thought this was needed to overcome the gitignore entry https://github.com/donejs/donejs/blob/master/.gitignore#L33 ... I remember having issues with the docs not being updated because I missed this step.